TundraTD/Assets/Scripts/Spells/SpellClasses/StoneGrenadeSpell.cs

```diff
@@ -1,14 +1,52 @@
+using System.Collections;
+using System.Collections.Generic;
+using Level;
+using Mobs.MobEffects;
+using Mobs.MobsBehaviour;
 using UnityEngine;
 
-namespace Spells
+namespace Spells.SpellClasses
 {
     public class StoneGrenadeSpell : MagicSpell
     {
+        [SerializeField] private GameObject stone;
+        [SerializeField] private float liftTime;
+        [SerializeField] private float lifeTime;
+        [SerializeField] private ParticleSystem blowParticle;
+
+        private readonly Collider[] _grenadeExplosionResults = new Collider[100];
+        
+        private const int MobLayer = 1 << 8; 
         public override BasicElement Element => BasicElement.Earth | BasicElement.Air;
 
-        public override void ExecuteSpell(RaycastHit hitInfo)
+        public override void ExecuteSpell(RaycastHit hitInfo) => StartCoroutine(ExecuteStoneLifting(hitInfo.point));
+        
+        private IEnumerator ExecuteStoneLifting(Vector3 position)
         {
-            throw new System.NotImplementedException();
+            var instantiatedStone = Instantiate(stone, position, Quaternion.identity, transform);
+
+            var time = 0f;
+            while (time < liftTime)
+            {
+                var lerpedY = Mathf.Lerp(0, 1, time / liftTime);
```

Comment:
Возможно, имеет смысл назвать переменную понятнее
```suggestion
                var interpolatedY = Mathf.Lerp(0, 1, time / liftTime);
```